src/dreyfus_index_updater.erl

```diff
@@ -31,6 +31,7 @@ update(IndexPid, Index) ->
     erlang:put(io_priority, {view_update, DbName, IndexName}),
     {ok, Db} = couch_db:open_int(DbName, []),
     try
+        {ok, ExcludeIdRevs} = purge_index(Db, IndexPid, Index),
```

Comment:
@chguocloudant Thanks for explanation, your current code makes more sense now.  Please keep it.